Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM python:2.7.11-alpine
+
+RUN pip install requests
+
+ADD . /src
+
+# UDP
+EXPOSE 1812
```

Comment:
I played with this a bit. It definitely needs the "/UDP" suffix. But I still could not get the networking to work without the "--net host" argument. Must be missing something.